Services/SymfonyVersionHelper.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Cron\CronBundle\Services;
+
+use AppKernel;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpKernel\Kernel;
+
+class SymfonyVersionHelper
+{
+	private $container;
+
+	public function __construct(Container $container)
+	{
+		$this->container = $container;
+	}
+
+	public function getRootDir()
+    {
+        if(Kernel::MAJOR_VERSION > 4)
+        {
+	        return $this->container->getParameter('kernel.root_dir');
+        }else{
+	        return $this->container->getParameter('kernel.root_dir');
```

Comment:
here should be kernel.project_dir i guess?

Reply:
Yes line 22. Copy and pasted line then forgot to change the parameter. My bad. Do you want me fix and create a new PR?

Reply:
Bty I didn't run unit tests just created a bare SF4 project and checked it.